lib/master.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * @file In order to load huge volumes of polygon data into memory without
+ *    breaking Node (due to its low heap-size limits), the package forks a
+ *    worker process per polygon layer/shapefile. This module contains
+ *    functions for initializing them and searching them.
+ */
+
+'use strict';
+
+var childProcess = require( 'child_process' );
+var peliasConfig = require( 'pelias-config' );
+var path = require( 'path' );
+var through = require( 'through2' );
+var logger = require( 'pelias-logger' ).get( 'admin-lookup:master' );
+
+/**
+ * For every desired administrative layer, specifies its name, the partial path
+ * of the corresponding Quattroshapes file, and the properties to extract from
+ * it.
+ */
+var quattroAdminLevels = [
```

Comment:
I'm not dead set on making those part of the config, but it just seemed to fall short of the intention. It looks like config that's been hardcoded in the implementation and it's just odd. It's not necessary to belabor this point any further.

Just mentioning in the README that the code relies on the Quattro path still isn't very helpful for someone that doesn't have a sample config file on-hand. This requires them to search the code to see where and how config is generated and then which fields are being used, or go into node_modules/pelias-config to search for the setting in question. If the README just mentions exactly which config is needed, it would go a long way. 

```javascript
{
  "imports": {
    "quattroshapes": {
      "datapath": "/path/to/Quattroshapes/simplified/"
    }
  }
}
```